packages/node_modules/@webex/internal-plugin-feature/src/feature.js

```diff
@@ -37,6 +37,22 @@ const Feature = WebexPlugin.extend({
     return Promise.resolve(feature.value);
   },
 
+  /**
+   * Issues request to server to set a value for a feature toggle.
```

Comment:
is this jsdoc correct?

Reply:
oops forgot to update. thanks for catching